examples/multi-brand-identity/src/styles/app.css

```diff
@@ -1,19 +1,52 @@
+/**
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

Comment:
This file was added in `2022` hence the year. There are several other such files in the PR.
Reviewers please note :)